layouts/shortcodes/home/cncf.html

```diff
@@ -1,5 +1,13 @@
-{{ $src := "https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/color/cncf-color.png" -}}
+{{ $src :=
+"https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/color/cncf-color.png"
+-}}
 <section class="home--top-section">
-  <h4>Notary project is a <a href="https://cncf.io">CNCF</a> incubating project</h4>
-  <img class="cncf-logo" src="{{ $src }}" alt="Cloud Native Computing Foundation logo">
+  <h4 class="in-project">
+    Notary project is a <a href="https://cncf.io">CNCF</a> incubating project
```

Comment:
```suggestion
    Notary Project is a <a href="https://cncf.io">CNCF</a> incubating project
```

Reply:
@sanjayk0508  Please note the project brand and name change in https://github.com/notaryproject/.github/issues/35